xds/internal/xdsclient/clientimpl_dump.go

```diff
@@ -37,11 +37,11 @@ func (c *clientImpl) dumpResources() *v3statuspb.ClientConfig {
 
 // DumpResources returns the status and contents of all xDS resources.
```

Comment:
This docstring should be updated to include the fact that `DumpResources` uses xDS clients from the default pool.

Reply:
Done